doc/api/util.md

```diff
@@ -392,6 +392,9 @@ changes:
   - version: REPLACEME
     pr-url: https://github.com/nodejs/node/pull/24852
     description: The `getters` option is supported now.
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: Internal properties do not leak anymore.
```

Comment:
I think this should go above the other one (so they appear in order when #24852 is released as a semver-minor commit); also, this description doesn’t provide a lot of information…

```suggestion
    description: Internal properties no longer appear in the context argument 
                 of a custom inspection function.
```